installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc

```diff
@@ -1,10 +1,17 @@
 [id="ipi-install-configuration-files"]
 = Configuration files
 :context: ipi-install-configuration-files
+:release: 4.6
```

Comment:
Hey @johnwilkins - what is the use of this attribute here? Unless I am misunderstanding, AsciiBinder provides the current version using the product-version attribute.

Reply:
@vikram-redhat in baremetal-deploy we use that value to define the version as we do use one set of docs for all releases and we use it in conditionals inside the code to show/hide the relevant sections

Reply:
Hi @iranzo - within OpenShift docs, you don't need this attribute. Instead use `{product-version}` wherever you want to use this and that will return, in the case of these docs, 4.6 and you can use that for conditionals like you would normally.